pulumi/__main__.py

```diff
@@ -0,0 +1,54 @@
+import pulumi_aws as aws
+from infra import dynamodb
+from infra.service_queue import ServiceQueue
+
+import pulumi
+
+PREFIX = pulumi.get_stack()
+
+if __name__ == "__main__":
+
+    dynamodb_tables = dynamodb.DynamoDB(PREFIX)
+
+    buckets = (
+        "analyzer-dispatched-bucket",
+        "analyzer-matched-subgraphs-bucket",
+        "analyzers-bucket",
+        "engagement-ux-bucket",
+        "model-plugins-bucket",
+        "osquery-log-bucket",
+        "subgraphs-generated-bucket",
+        "subgraphs-merged-bucket",
+        "sysmon-log-bucket",
+        "unid-subgraphs-generated-bucket",
+    )
+
+    for bucket in buckets:
+        bucket_name = f"{PREFIX}-{bucket}"
+        bucket = aws.s3.Bucket(
+            bucket,
+            bucket=bucket_name,
+        )
+        pulumi.export(f"Bucket: {bucket_name}", bucket.id)
```

Comment:
shadowing the `bucket` name here with something of a different type really rubs me the wrong way 

maybe `for bucket_name_suffix in buckets:` or something

sidenote: if pulumi is well typed, would love to see typechecking on this!

Reply:
Ah, yes... I forgot to add typing; I'll take care of that!

Reply:
Also, this isn't directly related to your question, but the question of naming reminded me of it... what value do we get from naming a bucket (and all the other resources we deal with) with a given prefix? I ask because controlling the name that tightly (i.e., conflating logical names and physical names) can have complicating implications.

- https://www.pulumi.com/blog/infrastructure-as-code-resource-naming/
- https://www.pulumi.com/docs/intro/concepts/resources/#names

This isn't something to be addressed for this PR, but was something I was curious about.